includes/discovery/sensors/temperature/avaya-vsp.inc.php

```diff
@@ -0,0 +1,71 @@
+/**
+ * avaya-vsp.inc.php
+ *
+ * LibreNMS Temperature Sensor Discovery module for Avaya VOSS(VSP Operating System Software)
+ *
+ * Copyright (c) 2017 Daniel Cox <danielcoxman@gmail.com>
+ *
+ * This program is free software: you can redistribute it and/or modify it
+ * under the terms of the GNU General Public License as published by the
+ * Free Software Foundation, either version 3 of the License, or (at your
+ * option) any later version.  Please see LICENSE.txt at the top level of
+ * the source code distribution for details.
+ */
+     
+$high_limit = 55;
```

Comment:
Why set hard limits? If the device doesn't provide them then we calculate the high / low limits.